packages/muriel-base-components/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "muriel-base-components",
```

Comment:
A bit academic as long as we don't publish, but to set good precedent: maybe scope, i.e. `@automattic/muriel-base-components`?